### PR TITLE
prepare add basemap

### PR DESCRIPTION
**Pull request by matallo**
PR to the basemaps branch (which is still WIP, tests pending), where I kickstart the add basemaps button, and the modals

just to have a quick feedback before moving on

CR @viddo

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/components/mosaic/mosaic-add-item-view.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    );
+    return this;
+  },
+
+  _onMouseEnter: function () {
+    this.model.set('highlighted', true);
+  },
+
+  _onMouseLeave: function () {
+    this.model.set('highlighted', false);
```
Comment (viddo) [inline lib/assets/javascripts/cartodb3/components/mosaic/mosaic-add-item-view.js]: What's the purpose of this `highlighted` state?

**Comment by viddo** [comment]
> just to have a quick feedback before moving on

Commented on a minor thing, but apart from that what kind of feedback do you expect at this point?

File: lib/assets/javascripts/cartodb3/components/mosaic/mosaic-add-item-view.js (hunk 1 above)
Comment (matallo) [inline lib/assets/javascripts/cartodb3/components/mosaic/mosaic-add-item-view.js]: based on mosaic (and carousel, which this was inspired on) it's the hover state with which the element changes the style and the description under the title updates accordingly (i.e. here I'm highlighting Dark Matter, the same would be with the + button)

screen shot 2016-06-14 at 16 10 34 2

**Comment by matallo** [comment]
I'm already showing/hiding the modal, just the file structure (and passing that `this._modals` in the waterfall)

**Pull request closed without merging**